week7/PhotoInspiration/app/src/main/java/com/manuflowers/photoinspiration/ui/detail/DetailViewModel.kt

```diff
@@ -0,0 +1,30 @@
+package com.manuflowers.photoinspiration.ui.detail
+
+import androidx.lifecycle.ViewModel
+import com.manuflowers.photoinspiration.application.PhotoInspirationApplication
+import com.manuflowers.photoinspiration.data.PhotosInspirationRepository
+import com.manuflowers.photoinspiration.data.local.database.PhotoInspirationDatabase
+import com.manuflowers.photoinspiration.data.local.database.PhotosDao
+import com.manuflowers.photoinspiration.data.local.preferences.PhotoInspirationPreferences
+import com.manuflowers.photoinspiration.data.remote.networking.RemoteApiManager
+import com.manuflowers.photoinspiration.data.remote.networking.buildApiService
+
+class DetailViewModel: ViewModel() {
+    private val apiService by lazy { buildApiService() }
```

Comment:
All of these properties are private. Do you even need/use this ViewModel?

Reply:
I realized that in week 8 😅